packages/firestore/test/integration/api/validation.test.ts

```diff
@@ -800,6 +800,14 @@ apiDescribe('Validation:', persistence => {
       );
     });
 
+    validationIt(persistence, 'enum', db => {
+      const collection = db.collection('test') as any;
```

Comment:
nit: maybe do `'foo' as any` below instead of `as any` here to minimize the scope of the untyped code we're dealing with (and to make the code better self-document exactly what we're doing that is invalid).